src/TestingUtils/Microsoft.AspNetCore.Testing/ref/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -1,7 +1,7 @@
 <!-- This file is automatically generated. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Why is there a ref/ project for Microsoft.AspNetCore.Testing at all? Should see warnings about non-`$(IsAspNetCoreApp)` projects having ref/ projects (though those are likely buried in other warnings at the moment)

Reply:
Removed